web/src/components/project/project-stars.tsx

```diff
@@ -23,6 +23,10 @@ export const ProjectStars: FC<Props> = (props) => {
 
   const [localStarred, setLocalStarred] = useState(isStarred);
 
+  useEffect(() => {
+    setLocalStarred(isStarred);
+  }, [project]);
+
```

Comment:
And again here...

Reply:
For some reason, the key={projectInfo.digest} didn't work here, so I'm gonna keep the useEffect for this one. I think it's because the project page refreshes differently than namespace when updating a project sample table.